canonical-web-and-design/maas-ui
# Add top level URL support

Done:
- Allow multiple to level URLS (not just /settings).
- Add basic machine view.

QA:
- Load the app and you should be at /machines.
- Click on Settings in the top nav, it should take you to the settings.
